src/CMakeLists.txt

```diff
@@ -220,8 +220,8 @@ install( FILES ${CMAKE_BINARY_DIR}/schema.xsd
 ################################
 # Add python environment setup
 ################################
-# message(WARNING "Temporarily changing the geosPythonBranch to cusini/addFailedTestsLogProcessing")
-# set(GEOS_PYTHON_PACKAGES_BRANCH "cusini/addFailedTestsLogProcessing" CACHE STRING "" FORCE)
+message(WARNING "Temporarily changing the geosPythonBranch to cusini/remove-relative-paths")
+set(GEOS_PYTHON_PACKAGES_BRANCH "cusini/remove-relative-paths" CACHE STRING "" FORCE)
```

Comment:
switch this back once https://github.com/GEOS-DEV/geosPythonPackages/pull/87 is merged.